_opcodes/op023-bg2.html

```diff
@@ -10,7 +10,20 @@
 iwd2: 0
 pst: 0
 ---
-Sets the Morale stat of the targetted creature(s) to 10.
+Applies the modifier value specified by the 'Statistic Modifier' field in the style specified by the 'Type' field.
+<br/>Known values for 'Type' are:
+<br/>0   Cumulative Modifier -> Morale = Morale + 'Statistic Modifier' value
+<br/>1   Flat Value Modifier -> Morale = 'Statistic Modifier' value
+<br/>2   Percentage Modifier -> Morale = (Morale * 'Statistic Modifier' value) / 100
+
+<br/>Known values for 'Mode' are:
+<br/>0   BG2 mode -> It's hardcoded to fill the following values:
+<ul>
+  <li>param #1 = 10</li>
+  <li>param #2 = 1</li>
+  <li>m_done = 1 (internal field that immediately removes the effect from creature, no matter duration)</li>
```

Comment:
First, there is no mode parameter — or did you forget to list it?

Second, I think this is needlessly complex. You can just state it as it was before: In BG2 it was hardcoded to set the Morale stat of the targetted creature(s) to 10. Best moved to the note below, which is currently quite crappy.

Third, you're editing  the wrong file. If ees have a proper opcode like the other games, then just make its listing use that one. Change the bgee: 1 to 0 and vice-versa in the main file. It's bg2 that's the outlier.